packages/mux-player/src/template.ts

```diff
@@ -80,6 +83,11 @@ export const content = (props: MuxTemplateProps) => html`
     default-hidden-captions="${props.defaultHiddenCaptions}"
     forward-seek-offset="${props.forwardSeekOffset}"
     backward-seek-offset="${props.backwardSeekOffset}"
+    playbackrates="${props.playbackRates ?? false}"
+    hide-duration="${[...props.controlsList.values()].some(
+      (key) => key.search(/no(top|bottom|center)?duration/) === 0
```

Comment:
nice 😎 

`.values()` can be left off if you like. `AttributeTokenList`'s iterator returns that as well.

nit (non-blocking) to be even more strict w/ start/end  could use a `/^no(top|bottom|center)?duration$/.test(key)`

Reply:
cool might as well make those changes since they're easy and are good improvements.